dojo_theme/templates/iframe.html

```diff
@@ -45,6 +45,16 @@ <h1 class="inactive">No active challenge session; start a challenge!<h1>
 
 {% block scripts %}
 {{ super() }}
+{% if active and loadUrlAsync %}
+<script>
+    let response =  fetch("/pwncollege_api/v1/workspace").then(response => response.json()).then(data => {
```

Comment:
Somehow we should pass to the workspace endpoint what url we're looking for. You can pass this all the way through with a jina `{{ service }}`.

Tangentially, it might also make sense to rename this from iframe.html => workspace.html. If there is someone else using iframe.html besides a workspace-related feature (I don't think so?) then we'd want to make a copy.